interface/forms/vitals/templates/vitals/general_new.html

```diff
@@ -9,136 +9,31 @@
 *}
 <html>
 <head>
-{headerTemplate assets='datetime-picker'}
-
+{headerTemplate assets='datetime-picker|reason-code-widget'}
+<!--Update the version (v flag) to cache bust if you modify this file -->
+<script src="{$webroot|attr}/interface/forms/vitals/vitals.js?v=1" type="text/javascript"></script>
```

Comment:
rec using that standard cache bust mechanism. Twig has a function for this. And doing it in one other script in smarty via {{ V_JS_INCLUDES }} in one of the document templates. We should make a smarty function that returns this, though.

Reply:
I really, really was half tempted to convert everything to twig... but there is more that Jerry needs so... gotta get stuff done.  I'll make the smarty function.